django-stubs/test/runner.pyi

```diff
@@ -159,12 +182,23 @@ class DiscoverRunner:
     def get_databases(self, suite: TestSuite) -> set[str]: ...
     def run_tests(self, test_labels: list[str], extra_tests: list[Any] | None = ..., **kwargs: Any) -> int: ...
 
-def is_discoverable(label: str) -> bool: ...
+def try_importing(label: str) -> bool: ...
```

Comment:
It returns `tuple[bool, bool]`: https://github.com/django/django/blob/14917c9ae272f47d23401100faa6cefa8e1728bf/django/test/runner.py#L1097